### PR TITLE
[WIP] Create slack service

### PR DESCRIPTION
**Pull request by callemand**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [X] If your changes affects code, did your write the tests?
- [X] Are tests passing? (`npm test` on both front/server)
- [X] Is the linter passing? (`npm run eslint` on both front/server)
- [X] Did you run prettier? (`npm run prettier` on both front/server)
~~- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)~~
~~- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/gladys-4-docs) and the [website](https://documentation.gladysassistant.com).~~
~~- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.json`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).~~

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change
Add Slack service link telegram
Screenshot 2019-11-11 at 20 21 13
Screenshot 2019-11-11 at 20 21 02

### CODE REVIEW

File: server/services/slack/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "gladys-telegram",
```
Comment (hotfix31) [inline server/services/slack/package.json]: maybe typo on package name ?

Comment (callemand) [reply server/services/slack/package.json]: Yes, fixed now

**Comment by Pierre-Gilles** [comment]
Hey! Thanks for this PR :) I guess it's still work in progress? 

If yes, can you put [WIP] in the title of this PR so it's easier for me?

**Comment by callemand** [comment]
> Hey! Thanks for this PR :) I guess it's still work in progress?
> If yes, can you put [WIP] in the title of this PR so it's easier for me?

Hi, 
For me the work is done. But I don't know the next step for a PR.

**Comment by Pierre-Gilles** [comment]
Thanks for the PR!  

> Hi,
For me the work is done. But I don't know the next step for a PR.

It seems there is just a small unit test missing, the diff coverage is -0.29% !

You'll see the detail here:

https://codecov.io/gh/GladysAssistant/Gladys/compare/09c873259660044e905c3f18ba71b6a6689f5f81...070da3a066b799697246bc9a9f599ca39f3c99d1/diff

Can you have a look? :)

**Comment by Pierre-Gilles** [comment]
Hi! What's the status for this PR? 

I'm putting this as "Work in Progress" (WIP) in the meantime

**Pull request closed without merging**